### ISSUE TITLE
Its possible to use with amplify?

### ISSUE THREAD

**Issue by berzavlu** [ava/use-http#340]
Hello i read the documentation and provider its the way, but amplify cognito send their own API library with their methods, but its possible to use with this library to use the hook? i mean with their own security properties on the header request from amplify

**Comment by alex-cory** [comment]
I'm sure there is. I haven't used amplify in a while. Are you saying you want to use the data returned from amplify in the headers for `useFetch`?

**Comment by berzavlu** [comment]
yes, because amplify provides methods POST, GET, etc. but without hooks like useFetch loading, error, response, data. It could be nice to use use-http library with amplify provider
